app/src/org/commcare/views/EntityDetailView.java

```diff
@@ -111,26 +115,26 @@ public EntityDetailView(Context context, Detail d, Entity e,
                             int index, int detailNumber) {
         super(context);
 
-        detailRow = (LinearLayout)View.inflate(context, R.layout.component_entity_detail_item, null);
-        label = (TextView)detailRow.findViewById(R.id.detail_type_text);
-        spacer = (TextView)detailRow.findViewById(R.id.entity_detail_spacer);
-        data = (TextView)detailRow.findViewById(R.id.detail_value_text);
+        detailRow = (LinearLayout) View.inflate(context, R.layout.component_entity_detail_item, null);

```

Comment:
Thanks. Was able to figure this out. Reformatted the code with cast spacing removed.